pages/services/spinnaker/0.3.0-1.9.2/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: layout.pug
+navigationTitle: Spinnaker 0.3.0-1.9.2
+title: Spinnaker 0.3.0-1.9.2
+menuWeight: 50
+excerpt: Documentation for DC/OS Spinnaker 0.3.0-1.9.2
+model: /services/spinnaker/data.yml
+render: mustache
+featureMaturity:
+enterprise: false
+---
+
+DC/OS {{ model.techName }} Service is an autonomous service that makes it easy to deploy and operate {{ model.serverName }} on Mesosphere [DC/OS](https://mesosphere.com/product/). For more information on Spinnaker, see the [Spinnaker documentation](https://github.com/spinnaker/spinnaker).
+
+**Note:** The DC/OS Spinnaker service currently only works with **DC/OS Enterprise**.
+
+## Benefits
+DC/OS {{ model.techName }} offers the following benefits:
+1. Based on the battle-proven DC/OS framework/operator SDK underpinning all DC/OS-certified frameworks
+2. One click install
+3. Automated failure recovery
+4. Automated configuration changes
+5. Automated scaling
+6. High availability
+7. TLS support
+8. Strict mode support
+9. Rack/Zone awareness
+
+## DC/OS {{ model.techName }}'s main features are:
+1. Deliver containers to DC/OS and Kubernetes
+2. Delivery pipelines
+3. Triggers - dockerhub, github, jenkins, ...
```

Comment:
are these 2 lists exhaustive?

Reply:
spinnaker has for like triggers and notifications many options, and those lists also change since new plugins get developed, so just giving a few samples here